### ISSUE TITLE
find fail whit newest mongo pecl

### ISSUE THREAD

**Issue by fxgit** [ichikaway/cakephp-mongodb#123]
I used this plugin for weeks and everything is ok but recientement install everything on a new server and the login fail.

install php mongo using the command

```
sudo pecl install mongo
```

but the app fail when use find method, the last records return: array( 'ModelName' => null)

create a info page

```
<?php phpinfo(); ?>
```

check that the version of mongo is 1.6 but in development server work with 1.5.8

```
#uninstall mongo 1.6.0
sudo pecl uninstall mongo

#install previus version
sudo pecl install mongo-1.5.8
```

**Comment by abhilashlohar** [comment]
I think you are using ubuntu.
try following:-
sudo apt-get install php5-dev php5-cli php-pear
sudo pecl install mongo



you can check version by
sudo pecl install mongo
sudo pecl install mongo

**Comment by abhilashlohar** [comment]
I think you should use an
`<addr>` element here instead.

**Comment by fxgit** [comment]
yes i'm using ubuntu server 14.04.1 LTS. I use these command and the problem was still, the problem was cakephp-mongodb dont work with the lastes version of  [The MongoDB PHP driver](http://php.net/manual/en/mongo.installation.php) installed via pecl

**Comment by abhilashlohar** [comment]
as i got you, you want to install web-server,Install MongoDB, php MongoDB Driver and RockMongo GUI on Ubuntu
just check this link it may be help-full for you.
`http://www.techspritz.com/install-mongodb-php-mongodb-driver-and-rockmongo-gui-on-ubuntu/`

**Comment by willianpts** [comment]
This is really a bug in this plugin. I've upgraded my [php-mongo driver to 1.6](https://github.com/mongodb/mongo-php-driver/tree/1.6.0) and the queries stopped working. They return an empty array instead of the results.

Downgrading php-mongo to 1.5.8 solves the issue.

**Comment by fxgit** [comment]
@abhilashlohar i'm reporting a bug in this plugin that isn't compatible con php-mongo driver 1.6. and the temporal fix is downgrade to 1.5.8
